### DISCUSSION TITLE
네이버 항공권 사이트 진입 불가 오류 문의

### DISCUSSION THREAD

**Discussion by sec123r** [List-KR/community#74] [AdGuard]
![image](https://github.com/user-attachments/assets/868c94bb-e2be-407b-b551-071a5a85f020)

네이버 -> 항공권 검색 -> 가격 조회 후 예매 페이지로 넘어갈 때
"사이트에 연결할 수 없음" 과 같은 오류가 뜨면서 페이지에 접속할 수 없는 오류가 있습니다.

DNS예외조건이나 사용자 규칙에 설정하고 접속해봐도 똑같은데 뭐가 문제인지 모르겠습니다 ㅠㅠ

**Comment by piquark6046** [comment]
네이버 -> 항공권 검색 단계에서의 예제 URL를 주실 수 있으시나요?

**Reply by sec123r** [reply]
https://m-flight.naver.com/
(모바일)

모바일 및 PC환경 모두 이쪽에 들어가서 테스트 해보았습니다. 

1. 아무 항공편이나 검색 (김포 ㅡ 제주, 성인1)
2. 이후 나오는 항공편 클릭 (여기까지는 문제없음)
3. 이후 예매를 위해 여행사 클릭시 (하나투어 같은) 오류가 발생합니다.



**flight.naver 도 그렇고 tivan.naver 도 그렇고 모두 타사이트 적용할때처럼 예외 설정 해봤는데, 안먹히더라구요ㅠ![Screenshot_20241205_181337_NAVER.jpg](https://github.com/user-attachments/assets/a68a12b1-e3d7-4e1b-8f11-266dbbadab8c)

![Screenshot_20241205_181349_NAVER.jpg](https://github.com/user-attachments/assets/07d4875c-914f-4e6b-959e-a52419fc684c)

![Screenshot_20241205_181341_NAVER.jpg](https://github.com/user-attachments/assets/9689a7ae-4efd-40a0-8a3a-d634fa25d58e)

**Reply by piquark6046** [reply]
https://github.com/AdguardTeam/AdGuardSDNSFilter/pull/1801 수정하기 패치를 남겼습니다
